### DISCUSSION TITLE
Full name for user account

### DISCUSSION THREAD

**Discussion by Marc-Proux** [docker-mailserver/docker-mailserver#4041] [Q&A] [answered]
Hello,

I would like to know if it's possible to create an email address that automatically includes a full name in the email client (like Outlook). The aim is for the recipient to see the full name of the sender rather than just the email address from my server.

Thank you in advance.

**Comment by glime-ai** [comment, marked as answer]
**Please note** that this response was created using generative AI, leveraging previously available information from the repository. This information may be helpful. However, it is our strong recommendation that you carefully consider the information before blindly applying it.

**This response is generated only once - you will not be able to interact with the glime-ai.**

---

Hello,

Yes, it is possible to include a full name in the email client so that the recipient sees the full name of the sender rather than just the email address. However, this is typically handled by the email client (like Outlook) rather than the mail server itself.

In most email clients, you can set a "Display Name" or "Full Name" when configuring the email account. This name will be shown to recipients instead of just the email address. Here’s a general way to do it in Outlook:

1. Open Outlook and go to the "File" menu.
2. Select "Account Settings" and then "Account Settings" again.
3. Choose the email account you want to edit and click "Change".
4. In the "User Information" section, you can set the "Your Name" field to the full name you want recipients to see.
5. Save the changes.

If you are using Docker Mailserver, the server itself does not handle the display name; it only handles the email address. The friendly name is managed by the email client when submitting an email for the server to send.

For more detailed instructions, you might want to refer to the documentation of your specific email client.

This comment was generated by [Glime](https://glimelab.ai/?utm_source=0d5b86fb7eea4a54acac405212c29aba&utm_medium=github).

**Comment by polarathene** [comment]
> The aim is for the recipient to see the full name of the sender rather than just the email address from my server.

Nothing to do with DMS AFAIK, your mail client does that part for you. How are you sending the mail? It should be configurable.

It's apparently possible via `smtp_header_checks` and the `REPLACE` rule in a `PCRE` table. That approach seems to be [discouraged though by the Postfix devs](https://www.mail-archive.com/search?l=postfix-users@postfix.org&q=subject:%22Re%3A+%22Display-name%22+rewriting%22&o=newest&f=1), to which there is no other alternative shared when you want to manipulate the _sender address_ to show a real/display/pretty name in addition to the mail address itself.

If you're comfortable taking the information I gave above to configure it in Postfix, you may do so but it's not something DMS supports officially, and you are better off to configure such via your mail client AFAIK.

**Reply by Marc-Proux** [reply]
Thank you for the response. 
Yes, it is configurable directly in the client (though not in all of them), but I was wondering if it could be done automatically. It seems that it cannot.
Thank you for the help!
